self-serve-storage/R/Storage Examples.ipynb

```diff
@@ -8,7 +8,7 @@
     "\n",
     "For this code to work, you need to have set the option to add Minio credentials when you started the server\n",
     "\n",
-    "![Minio Credentials Option](kubeflow-minio-option.png)\n",
+    "![Minio Credentials Option](../../_images/kubeflow-minio-config.png)\n",
```

Comment:
I thought about this, but I think it's actually a bad idea because it prevents users from being able to copy folders over as templates in new projects. I think we want each folder to be self-contained (even if this causes duplication)